src/components/IconMapper/dynamicFontAwesomeImports.js

```diff
@@ -1,41 +1,41 @@
-import { faCubes } from "@fortawesome/free-solid-svg-icons";
-import { faPersonWalkingLuggage } from "@fortawesome/free-solid-svg-icons";
-import { faObjectGroup } from "@fortawesome/free-solid-svg-icons";
-import { faEnvelopeOpenText } from "@fortawesome/free-solid-svg-icons";
-import { faDatabase } from "@fortawesome/free-solid-svg-icons";
-import { faHdd } from "@fortawesome/free-solid-svg-icons";
-import { faCloudArrowDown } from "@fortawesome/free-solid-svg-icons";
-import { faServer } from "@fortawesome/free-solid-svg-icons";
-import { faUserShield } from "@fortawesome/free-solid-svg-icons";
-import { faUsers } from "@fortawesome/free-solid-svg-icons";
-import { faWarehouse } from "@fortawesome/free-solid-svg-icons";
-import { faBook } from "@fortawesome/free-solid-svg-icons";
-import { faBookmark } from "@fortawesome/free-solid-svg-icons";
-import { faGavel } from "@fortawesome/free-solid-svg-icons";
-import { faTerminal } from "@fortawesome/free-solid-svg-icons";
-import { faLock } from "@fortawesome/free-solid-svg-icons";
-import { faGears } from "@fortawesome/free-solid-svg-icons";
-import { faScrewdriverWrench } from "@fortawesome/free-solid-svg-icons";
-import { faShield } from "@fortawesome/free-solid-svg-icons";
+import { faCubes } from '@fortawesome/free-solid-svg-icons';
```

Comment:
i thought this file was part of the `prettierignore` ?

Reply:
@addetz It will be. I just need to update the branch 😄